src/contexts/WalletContext/services.ts

```diff
@@ -189,24 +232,26 @@ export async function getWalletRiskAssessment(walletId: string) {
     const result = await fetch('https://api.trmlabs.com/public/v2/screening/addresses', {
       method: 'POST',
       headers: {
-        'Accept': 'application/json',
+        Accept: 'application/json',
         'Content-type': 'application/json',
-        Authorization: 'Basic ' + btoa(process.env.REACT_APP_TRM_API_KEY || '')
+        Authorization: 'Basic ' + btoa(process.env.REACT_APP_TRM_API_KEY || ''),
```

Comment:
```suggestion
        'Authorization': 'Basic ' + btoa(process.env.REACT_APP_TRM_API_KEY || ''),
```

keep consistency?

Reply:
If I make your suggested change, prettier (or linting; not sure which) undoes it automatically. (I guess the difference is the '-' in 'Content-type'.) So leaving this as is.

Reply:
lol :D gonna fix that one of these days! thanks for the headsup!